pkg/minikube/image/image.go

```diff
@@ -19,6 +19,7 @@ package image
 import (
 	"context"
 	"fmt"
+	"github.com/google/go-containerregistry/pkg/v1/mutate"
```

Comment:
Should probably group this import with the rest of the GitHub imports below instead

Reply:
good point. fixed